molgenis-core-ui/src/main/resources/templates/molgenis-header-tracking.ftl

```diff
@@ -16,10 +16,10 @@
             m.parentNode.insertBefore(a, m)
         })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
         <#if app_settings.googleAnalyticsTrackingId??>
-            ga('create', '${app_settings.googleAnalyticsTrackingId?html}', 'auto');
+            ga('create', '${app_settings.googleAnalyticsTrackingId?html}', 'auto', {'cookieFlags': 'SameSite=None; Secure'});
```

Comment:
Secure won't work over http (i.e. development machine) but is required with SameSite=None. I think we don't need SameSite=None and SameSite=Lax will work just fine. Then again, we probably don't need ga on dev machines either, unless when we are debugging.